### PR TITLE
Chopsticks in C++

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Code added in cp(competitive programming) folder

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6401

### CODE REVIEW

**Comment by gayatri517** [comment]
@ankitaggarwal23 Please review this PR.

**Comment by gayatri517** [comment]
@ankitaggarwal23 @HarshCasper Please review my PR

File: C-Plus-Plus/cp/chopsticks.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        {
+            i++;
+        }
+    }
+    return res;
+}
+
+int main()
+{
+    int n,d,a;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/chopsticks.cpp:31]: Use a good naming convention for all three

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@gayatri517 Small changes are required

**Pull request closed without merging**